coqpyt/tests/proof_file/expected/list_notation.yml

```diff
@@ -20,7 +20,6 @@ proofs:
             line: 4
             character: 7
         context:
-          # FIXME issue #24
```

Comment:
Why did you remove this?

Reply:
A FIXME was added to the actual test making this comment redundant.